templates/frontend/base.html

```diff
@@ -1,39 +1,110 @@
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
-    <meta name="keywords" content="Koggala Lake, Chari Lake, Sri Lanka, Transcend Beyond Relaxation And Luxury, Galle Fort, Whale watching, Unawatuna Beach, Yala, Uda-walawe, Cinnamon Island" />
-	<meta name="description" content="Set beside the verdant greenery, lush vegetation and the spectacular Koggala Lake, Galle, CharÌ Lake offers an experience that is like no other. Just a 90 minute drive from Colombo it can also be accessed in 15 minutes by a domestic flight to Koggala. This contemporary boutique hotel consists of exquisitely and exclusively designed 10 rooms and one suite." />
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      name="keywords"
+      content="Koggala Lake, Chari Lake, Sri Lanka, Transcend Beyond Relaxation And Luxury, Galle Fort, Whale watching, Unawatuna Beach, Yala, Uda-walawe, Cinnamon Island"
+    />
+    <meta
+      name="description"
+      content="Set beside the verdant greenery, lush vegetation and the spectacular Koggala Lake, Galle, CharÌ Lake offers an experience that is like no other. Just a 90 minute drive from Colombo it can also be accessed in 15 minutes by a domestic flight to Koggala. This contemporary boutique hotel consists of exquisitely and exclusively designed 10 rooms and one suite."
+    />
     <title>{% block title %}{% endblock %}Chari Lake Hotel</title>
 
-    <link rel="icon" href="{% static 'favicon.jpg' %}" type="image/jpg">
-    <link rel="stylesheet" href="{% static 'css/style.min.css' %}" type="text/css"/>
+    <link rel="icon" href="{% static 'favicon.jpg' %}" type="image/jpg" />
+    <link
+      rel="stylesheet"
+      href="{% static 'css/style.min.css' %}"
+      type="text/css"
+    />
     {% block css %}{% endblock %}
 
-    <link rel="preconnect dns-prefetch" href="https://cdn.iconicto.com" crossorigin>
-    <link rel="preconnect dns-prefetch" href="https://fonts.googleapis.com" crossorigin>
-    <link rel="preconnect dns-prefetch" href="https://kit-free.fontawesome.com" crossorigin>
-    <link rel="preconnect dns-prefetch" href="https://www.google-analytics.com" crossorigin>
-</head>
-<body>
-{% include 'frontend/partial/navbar.html' %}
-    {% block content %}{% endblock %}
-{% include 'frontend/partial/footer.html' %}
-    {% block js %}{% endblock %}
+    <link
+      rel="preconnect dns-prefetch"
+      href="https://cdn.iconicto.com"
+      crossorigin
+    />
+    <link
+      rel="preconnect dns-prefetch"
+      href="https://fonts.googleapis.com"
+      crossorigin
+    />
+    <link
+      rel="preconnect dns-prefetch"
+      href="https://kit-free.fontawesome.com"
+      crossorigin
+    />
+    <link
+      rel="preconnect dns-prefetch"
+      href="https://www.google-analytics.com"
+      crossorigin
+    />
+  </head>
```

Comment:
These links need to be present in the `<head>` if you have these in after content, I think it's gonna cache the links after all the content because HTML is parsed top to bottom

Reference: <https://web.dev/link-prefetch/>